### ISSUE TITLE
Vault stops updating dynamically in web vault when browser extension is opened in own window

### ISSUE THREAD

**Issue by jikquantopian** [bitwarden/web#334]
When I click the button to detach the browser extension from the URL bar and move it into its own window and I have the web vault open as well and then I make changes to items in my vault (in particular, moving them out of folders), the web vault doesn't update to reflect the changes.

At least, I _think_ that's what's going on. I can tell you for certain that I've got my web vault open right now and it's not updating and I've also got the popped-out extension window open. But I suppose those two facts could be independent.

**Comment by kspearrin** [comment]
Detaching the extension window shouldn't have anything to do with it. If you refresh the web vault, do live syncs from other application begin to work after that?

**Comment by jikquantopian** [comment]
I guess you're right that they're independent.

Yes, if I reload it resumes updating, but then intermittently it stops again.

**Comment by kspearrin** [comment]
If you open dev tools in the browser, under the network tab you should see the websocket connection. Do you see it dropping out and not reconnection or anything abnormal like that? It is normal for it to disconnect from the websocket connection after a certain timeout of inactivity, but it should reconnect again whenever you interact with the web vault window.

**Comment by jikquantopian** [comment]
Ah, OK, so that's the issue. I've got the web vault window open but I'm not interacting with it, so it stops getting updates. If that's by design, I am not sure this is a "bug," but it was certainly unexpected behavior for me. I mean, I could see the window right there on my screen not getting updated when I made changes in a different window that should have showed up in the one I was looking at. That's definitely not intuitive.
